charts/merlin/templates/service-monitor.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.monitoring.enabled }}
```

Comment:
iirc., There are two monitors added for merlin, the app monitor and there is a merlin-tracker monitor. Do we need to add that too?

Reply:
I think it's deployed separately from merlin